djcelery/models.py

```diff
@@ -182,6 +184,8 @@ class PeriodicTasks(models.Model):
     ident = models.SmallIntegerField(default=1, primary_key=True, unique=True)
     last_update = models.DateTimeField(null=False)
 
+    no_changes = False
```

Comment:
I try to re-use the same trigger for PeriodicTask when IntervalSchedule or CrontabSchedule has some change.

So, without this, you need to restart de celery beat to reload configuration of CrontabSchedule or IntervalSchedule, or you need to trigger some modification on PeriodicTask to reload configuration.